whylog/client/__init__.py

```diff
@@ -1,6 +1,28 @@
-class WhylogClient(object):
-    def __init__(self, **kwargs):
+from abc import ABCMeta, abstractmethod
+import os
+
+from whylog.config import YamlConfig
+
+
+class AbstractClient(object):
+
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def get_cause(self):
+        pass
+
+
+class WhylogClient(AbstractClient):
+    """
```

Comment:
Odd docstring identation - maybe we can use something more popular in other libraries?